privacyidea/static/components/token/views/token.import.html

```diff
@@ -69,12 +69,27 @@ <h3 translate>Import tokens from a file</h3>
         </p>
         <input class="form-control"
                placeholder="{{ 'Pre Shared Key' | translate }}"
-               ng-keypress="form.password =''"
+               ng-keypress="form.password = ''"
                ng-model="form.psk" minlength="0" maxlength="32">
         <input class="form-control"
                placeholder="{{ 'Password' | translate }}"
                ng-keypress="form.psk = ''"
                ng-model="form.password">
+        <div></div>
+        <div class="mt-5px">
+            <p class="help help-block" translate>
+                You can choose how the authenticity the stored tokens should be verified.
```

Comment:
I would say

    "You can choose how the authenticity of the tokens to be imported should be verified".

I might be otherwise confusing, that the "stored tokens" would be the token in the privacyIDEA database.